docs/conf.py

```diff
@@ -161,9 +78,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'CAMP'
-copyright = u'2019, Tom Scogland'
-author = u'Tom Scogland'
+project = u'Camp'
+copyright = u'2016-2024, Lawrence Livermore National Security, LLNS'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'2024.02'
```

Comment:
```suggestion
version = u'2024.07'
```

To match the last released version

Reply:
I pushed a change to address this.